src/angular/hq/src/app/services/hq.service.ts

```diff
@@ -19,6 +19,12 @@ export class HQService {
     );
   }
 
+  getClientV1(request:  Partial<GetClientRequestV1>) {
```

Comment:
We don't need to keep these 2 as separate, I don't mind if we just use Partial<T> as a convention for the request objects.